e2e/bitcoin-spv-failover.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -e # Exit immediately when the script failed
+echo "Starting bitcoind node and bitcoin-lightclient..."
+make bitcoind-init
+cd contrib/
+docker compose up -d
+cd ../
+
+echo "Starting bitcoin-spv relayer..."
+go build ./cmd/bitcoin-spv/
+./bitcoin-spv start --config ./sample-bitcoin-relayer.yml 2>stderr.log &
+RELAYER_PID_OLD=$! # gets pid of last background process
+echo "Started bitcoin-spv relayer with PID $RELAYER_PID_OLD"
+
+echo "Waiting for bitcoin-spv relayer to bootstrap..."
+sleep 10
+
+# make sure bitcoind node is up and running
+if ! docker exec -it bitcoind-node bitcoin-cli -regtest getblockchaininfo; then
+  echo "ERROR: failed to start bitcoind node"
+  exit 1
+fi
+
+# query the lightclient for the latest block
```

Comment:
this is not the light client. We need to call the LC on SUI

Reply:
Updated